docs/examples.rst

```diff
@@ -677,6 +677,37 @@ variables or context dimensions:
         ref: $@distro
 
 
+.. _anchors-aliases:
+
+Using anchors and aliases in Plans
+------------------------------------------------------------------
```

Comment:
I was thinking to slowly deprecate the `Examples` section and gradually move the content from example to the `Guide`. What about adding a new `Aliases` section to right after the [Elasticity](https://tmt.readthedocs.io/en/latest/guide.html#elasticity)? Seems this would be a similar type of hint how to maintain metadata efficiently.